src/evict/evict_lru.c

```diff
@@ -1980,7 +1980,7 @@ __evict_walk_tree(WT_SESSION_IMPL *session,
 		 * The goal here is that if trees become completely idle, we
 		 * eventually push them out of cache completely.
 		 */
-		if (WT_PAGE_IS_INTERNAL(page)) {
+		if (false && WT_PAGE_IS_INTERNAL(page)) {
```

Comment:
Mix it up: include internal pages regularly, and (with the `__evict_lru_cmp` change above), don't weight in favor of leaf pages.